src/backend/access/external/pxfheaders.c

```diff
@@ -158,6 +165,29 @@ static void add_tuple_desc_httpheader(CHURL_HEADERS headers, Relation rel)
 	pfree(formatter.data);
 }
 
+static void add_projection_desc_httpheader(CHURL_HEADERS headers, ProjectionInfo *projInfo) {
+	int i;
+	char long_number[32];
```

Comment:
Since we are converting `list_length(projInfo->pi_targetlist)`, we need an array thats 32 chars long to match the bit length of `int`.